### DISCUSSION TITLE
next/head title reset to the previous document.title value on tab switch

### DISCUSSION THREAD

**Discussion by hybeats** [vercel/next.js#17717] [Help]
# Bug report

## Describe the bug

next/head title reset to the previous document.title value on tab switch

We have 2 pages that use next/head tag with title value. 

## To Reproduce

On initial load everything works as expected but after going to another route (second page with another next/head title value) and after switching between browser tabs, document.title value resets to the previous one.

## Expected behavior

document title is correctly changing to the current value

## Screenshots

video
https://ttprivatenew.s3.amazonaws.com/pulse/hybeats/attachments/14899015/TinyTake08-10-2020-05-43-55.mp4

## System information

- OS: [macOS, Windows]
- Browser (if applies) [all browsers]
- Version of Next.js: [9.5.3]
- Version of Node.js: [e.g. 12.16.2]

**Comment by RickyTB** [comment]
Same thing is happening to me. Did you find a solution?

**Reply by kareemiLeaf** [reply]
if (process.browser) {
      window.addEventListener("focus", function () {
        document.title = title;
      });
      window.addEventListener("blur", function () {
        document.title = title;
      });
    }
